exercises/solutions/solution_02.py

```diff
@@ -1,26 +1,24 @@
-# Popular flights - hitting the Show more button leads to an increment of currently displayed popular flight
-# tiles/cards to be displayed
-def test_show_more_button_of_popular_flights_displays_increment_of_currently_shown_popular_flight_cards(page):
-    # 1. On the Kiwi.com website get the count of tiles/cards in the Popular flights section
-    # and store it into a variable
-    # 1.1. Open the kiwi.com website (https://www.kiwi.com/en/) and accept cookies
-    page.goto("https://www.kiwi.com/en/")
-    page.locator("[data-test='CookiesPopup-Accept']").click()
-    assert page.locator("text=Book cheap flights other sites simply can’t find.").is_visible()
+# Interacting with the regional settings and dropdown lists
 
-    # 1.2. Get count of tiles/cards in the Popular flights section and store it into a variable
-    initial_card_count = page.locator("[data-test=PopularFlights] [class*=PopularFlightCardstyled__Card]").count()
+def test_regional_settings_can_be_changed(page):
+	# 1. On the Kiwi.com website and open the regional settings modal
```

Comment:
Indentation should be half the size used here (same for `exercises/tasks/task_02.py`).